test/cases/vmware-ovf.sh

```diff
@@ -0,0 +1,238 @@
+#!/bin/bash
```

Comment:
I'm wondering if it would be simple enough just extend the `vmware.sh` script to accept argument whether we want to test VMDK or OVA import. It seems to me that big portion of the script is the same (except for the image type when building the image and the way the VM is created in VSphere)... 🤔  Basically something like we do with `api.sh`.

Reply:
I merged them, but i wonder if we should also run both the vmdk and the ova test on all RHELs. Since they're not that different.